x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/rollup/action/GetRollupJobsAction.java

```diff
@@ -254,7 +254,12 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             builder.field(CONFIG.getPreferredName());
             job.toXContent(builder, params);
             builder.field(STATUS.getPreferredName(), status);
-            builder.field(STATS.getPreferredName(), stats);
+
+            // special BWC handling for rollup
+            builder.startObject(STATS.getPreferredName());
+            stats.toUnwrappedXContent(builder, true);
```

Comment:
The problem I see here is that there is no way for users of rollup to migrate to the new response API. I think we need a parameter passed in through the `Params` argument to this method which the user can control to determine whether they ask for the new response format or the old one (with it defaulting to the old one. In fact, its possibly better for `IndexerJobStats.toXContent()` to get this parameter itself so the bwc is fully contained within that class?